CMakeLists.txt

```diff
@@ -133,14 +133,15 @@ SET (ORION_LIBS
     rest
     jsonParse
     jsonParseV2
-    apiTypesV2
+    rest
```

Comment:
Good idea to include the name of the function... in the case we already know it. If knowning it requires research that could delay merging this PR, I'd suggest jus to include a "neutral" comment without mentioning it.